src/frequenz/sdk/timeseries/data_pipeline.py

```diff
@@ -0,0 +1,115 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Create, connect and own instances of data pipeline components."""
```

Comment:
I feel this description doesn't really tell me anything about what this class can be used for

Reply:
I've added a few more lines,  but this is now also internal.  So end users who don't have enough context will no longer be exposed to it directly.